### ISSUE TITLE
super outdated crates.io

### ISSUE THREAD

**Issue by nbrr** [vitiral/artifact#267]
I tried to `cargo install artifact-app` in order to update artifact only to realize it tried to build v1.0.1 (which failed btw), and that the artifact crates available were super old. How come? Maybe you should just drop the `cargo install` from the tutorial.

I then realized that I originally just installed the binaries.

While I'm there, latest release v2.1.5 panics when I just invoke `art` with no argument, I guess it should probably display some help info.

**Comment by vitiral** [comment]
Yes, I havne't pushed to crates.io in a while. The main reason is because artifact uses two stacks (web-frontend and backend) which must both be built, and I have not (yet) been able to stuff them both in a build.rs file (relatedly, it currently requires cargo-web and mdbook).

The `art` panic issue has been resolved in master :smile:

**Comment by vitiral** [comment]
The tutorial needs a general overhaul and I haven't had time to do it since moving and starting a new job. It might be some time before I get to it.
